Let's follow the template.

ericgaspar opened the pull request.

ericgaspar committed:
Let's follow the template.

ericgaspar committed:
Create tests.toml

ericgaspar committed:
Update manifest.toml

yunohost-bot committed:
Auto-update READMEs

ericgaspar committed:
Update manifest.toml

ericgaspar committed:
Update manifest.toml

ericgaspar committed:
cleaning

ericgaspar committed:
Update install

ericgaspar committed:
Update nginx.conf

ericgaspar committed:
cleaning

yunohost-bot committed:
Auto-update READMEs

ericgaspar committed:
cleaning

ericgaspar committed:
Update config_panel.toml

ericgaspar committed:
cleaning

fflorent commented on scripts/install:
```diff
@@ -122,26 +105,22 @@ $data_dir/.venv/bin/umap create_superuser --username="$admin" --email="$(ynh_use
-yunohost service add --description $app $app
+ynh_config_add_systemd
+
+yunohost service add "$app" --description="A short description of the app" --log="/var/log/$app/$app.log"
```
```suggestion
yunohost service add "$app" --description="Create maps with OpenStreetMap layers" --log="/var/log/$app/$app.log"
```

fflorent commented on scripts/upgrade:
```diff
@@ -95,13 +89,18 @@ $data_dir/.venv/bin/umap check --deploy || true
+
+ynh_config_add_systemd
+
+yunohost service add "$app" --description="A short description of the app" --log="/var/log/$app/$app.log"
```
```suggestion
yunohost service add "$app" --description="Create maps with OpenStreetMap layers" --log="/var/log/$app/$app.log"
```

fflorent commented on config_panel.toml:
```diff
@@ -26,21 +26,6 @@ services = ["__APP__"]
         help = "Email address for error emails."
         bind = "admin_email:/home/yunohost.app/__APP__/settings.py"
```
À tester si ça (`default_from_email` et `admin_email`) fonctionne toujours dans le config panel.

Mais à supprimer dans tous les cas car l'upgrade supprime leur valeur.

fflorent commented on conf/systemd.service:
```diff
@@ -5,7 +5,7 @@ After=network.target redis.service postgresql.service
 User=__APP__
 WorkingDirectory=__DATA_DIR__/
-EnvironmentFile=__DATA_DIR__/env
+EnvironmentFile=__DATA_DIR__/.env
```
Adapter les chemins, tous ne le sont pas (dans `scripts/install` et dans `scripts/upgrade`)

fflorent committed:
Fix env path to source + use pushd / popd

We usually avoid using `cd` when packaging, we prefer using `pushd` so
we can use a queue of directories and go backward easily. Also the
indentation helps us to track in which directory we are.

fflorent committed:
Add meaningful service description when adding it

fflorent committed:
Clean variables that are use in config panel

These variables are not used anymore in config panel, and in _common.sh
they overwrite the user's choice during installation.
Also set email in upgrade so settings.py can substitute `__EMAIL__` with
it's value when running ynh_config_add

fflorent committed:
Remove data_dir no matter whether the user chose to purge

This is a problem that is tracked in
https://github.com/YunoHost-Apps/umap_ynh/issues/11

fflorent committed:
Remove commented boilerplate in tests.toml

fflorent committed:
Fix installation by reordering the scripts execution

collectstatic should be run before migrate

fflorent committed:
Remove useless debug_enabled var in settings.py
